src/pages/FirstPage.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from 'react';
+import { StepContext } from '../App';
+import styled from 'styled-components';
+import theme from '../styles/theme';
+
+const FirstPage = () => {
+  const { step, setStep } = useContext(StepContext);
+
+  return (
+    <Container>
+      <CardContainer>
+        <SubTitle>어르신 돌보미</SubTitle>
+        <Title>케어코디</Title>
+      </CardContainer>
+      <LandingButton onClick={() => setStep(step + 1)}>신청하기</LandingButton>
```

Comment:
```suggestion
      <LandingButton onClick={() => setStep({...step,number:0)}>신청하기</LandingButton>
```
step은 객체로 되어있고, `number` 값이 있어야 step 페이지가 띄우도록 작성되어있기 때문에 위와 같이 작성하셔야 합니다.

Reply:
네 수정했습니다!

Reply:
네 수정했습니다!